tests/web3/blake2.expected.json

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "jsonrpc": "2.0",
+    "id": 0,
+    "result": ""
+  },
+  {
+    "jsonrpc": "2.0",
+    "id": 1,
+    "result": ""
+  },
+  {
+    "jsonrpc": "2.0",
+    "id": 2,
+    "result": ""
+  },
+  {
+    "jsonrpc": "2.0",
+    "id": 3,
+    "result": ""
+  },
```

Comment:
If these correspond to error cases, any chance we can have a more sensible error message?

Reply:
Not sure if that would be worth the effort, since the function isn't supposed to be exposed on the RPC like this.